ios/Trifle/Sources/Digital Signature/Secure Enclave/SecureEnclaveDigitalSignatureKeyManager.swift

```diff
@@ -72,32 +73,53 @@ public class SecureEnclaveDigitalSignatureKeyManager
 
     // MARK: - Internal Methods (DigitalSignatureKeyManager)
 
-    internal func signingKey() throws -> SecureEnclaveSigningKey {
+    internal func createOrLoadSigningKey() throws -> SecureEnclaveSigningKey {
         let signingAlgorithm = Self.keyInfo.signingAlgorithm.attrs
 
-        guard try keyExists() else {
+        if try keyExists() {
```

Comment:
keep the guard statement here and call `loadSigningKey` at the end of the code block

Reply:
since we're initializing the keypair in the initializer, `signingKey()` can just be:
```
var keyRef: CFTypeRef?
        let preparedQuery = SecureEnclaveKeychainQueries.getQuery(with: tag, returnRef: true)
        SecItemCopyMatching(preparedQuery, &keyRef)

        return try SecureEnclaveSigningKey(keyRef as! SecKey, Self.keyInfo.signingAlgorithm.attrs)
```

and `loadSigningKey()` can go away